Assets/SdpTransform/SdpUtils/UnifiedPlanUtils.cs

```diff
@@ -51,6 +51,7 @@ public static List<RtpEncodingParameters> GetRtpEncodingParameters(MediaDescript
 
         foreach (var ssrc in ssrcs)
         {
+            if (ssrcToRtxSsrc.ContainsKey(ssrc)) ssrcToRtxSsrc.Remove(ssrc);

```

Comment:
In typescript you can directly set a key, but not here. Remove the earlier one before you add, otherwise you get exception